coalib/collecting/Importers.py

```diff
@@ -113,20 +113,16 @@ def _iimport_objects(file_paths, names, types, supers, attributes, local):
     :raises Exception: Any exception that is thrown in module code or an
                        ImportError if paths are erroneous.
     """
-    if (file_paths == [] or
-            (names == [] and
-             types == [] and
-             supers == [] and
-             attributes == [])):
-        raise StopIteration
+    if not (file_paths and (names or types or supers or attributes)):
```

Comment:
I double-negated it, and expanded one negation onto the whole expression^^
if you think I should keep the other version, then we should at least do
`if (not file_paths or (not names and not ...))`